### ISSUE TITLE
Public server down.

### ISSUE THREAD

**Issue by marcushat** [Project-OSRM/osrm-backend#2037]
router.project-osrm.com went down(503 errors) a few mins ago.

**Comment by daniel-j-h** [comment]
Yes, already working on it. By the speedy ticket I assume you rely on it?

**Comment by marcushat** [comment]
Not really, I've just been using it a lot this week for tests. Any estimate on when it'll be back?

**Comment by daniel-j-h** [comment]
Hopefully soon (TM). I would guess in the next 20-30 minutes, depending on if I can fix the issue or have to roll back to an older version.
